source/config/profileUpgradeSteps.py

```diff
@@ -101,7 +101,7 @@ def upgradeConfigFrom_5_to_6(profile: dict):
 		profile['UIA']['allowInMSWord'] = AllowUiaInMSWord.ALWAYS.value
 
 
-def upgradeConfigFrom_6_to_7(profile: dict):
+def upgradeConfigFrom_6_to_7(profile: Dict[str, str]) -> None:

```

Comment:
For some reason, this change prevents NVDA from starting on my system.

Reply:
You need to import `Dict` from `typing`